packages/vite/src/node/index.ts

```diff
@@ -111,7 +111,6 @@ export type {
   InferCustomEventPayload,
   InvalidatePayload
 } from 'types/customEvent'
-// [deprecated: use vite/client/types instead]
```

Comment:
This was deprecated by #9966 and reverted by #10243 but the comment was still there.